packages/ag-charts-enterprise/src/features/zoom/zoom.ts

```diff
@@ -199,7 +200,11 @@ export class Zoom extends _ModuleSupport.BaseModuleInstance implements _ModuleSu
     }
 
     private onDrag(event: _ModuleSupport.InteractionEvent<'drag'>) {
+        const state = this.ctx.interactionManager.state;
         if (!this.enabled || !this.paddedRect || !this.seriesRect) return;
+        if (state !== InteractionState.Default && state !== InteractionState.ZoomPan) return;
+
+        this.ctx.interactionManager.pushState(InteractionState.ZoomPan);
```

Comment:
nit: it's not just pan here, can also be selection or axis zoom, so perhaps a more generic `InteractionState.ZoomDrag`?

Reply:
Done. Renamed `ZoomPan` to `ZoomDrag`